packages/cordova-builders/utils/index.ts

```diff
@@ -171,7 +171,7 @@ export function prepareServerConfig(options: CordovaServeBuilderSchema, root: st
     let { input, output, ignore = [], glob } = asset;
     input = resolve(root, input).replace(/\\/g, '/');
     input = input.endsWith('/') ? input : input + '/';
-    output = output.endsWith('/') ? output : output + '/';
+    output = output?.endsWith('/') ? output : output + '/';
```

Comment:
Can you explain this change? If `output` can be undefined, then the resulting value with this change becomes `undefined/` vs. throwing an exception. 

I would not expect that `output` can be undefined.

Reply:
Without the ? I get a typescript error on the statement: 
"'Output' is possibly undefined." 
This is because Output is defined in AssetPatternClass as Output?: string.

Reply:
We should update the implementation to:
```ts
output = output?.endsWith('/') ? output : (output ?? '') + '/';
```

to handle the case when the `output` is `undefined`. I'm not aware of the edge case where it is unset, but this way it will at least fallback to resolve `/`.